src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped working");

```

Comment:
Similar to the doWork method, the message 'Truck stopped working' should be more informative. Include details that identify the specific Truck that has stopped working.